Warmup/assignment.md

```diff
@@ -8,3 +8,4 @@ Problem| solved | Date
 --- | --- | --- | 
 https://www.spoj.com/problems/DOL/ |  |  |  |
 
+:
```

Comment:
Remove this colon and put it in appropriate position.

Reply:
Changes Commited

Reply:
@nileshhazra, Let me look into it.!